### ISSUE TITLE
GoogleApi.Entities.Translate.Common.Enums.Language Kinyarwanda ["rw"] is missing

### ISSUE THREAD

**Issue by ewhoambra** [vivet/GoogleApi#220]
I've installed GoogleApi v. 3.10.3 in my C# project.

It seems that on February '20 Google implemented a new language named: "Kinyarwanda" ISO Code: RW.

Running this simple code throws exception:
`var lng = GoogleApi.GoogleTranslate.Languages.Query(new GoogleApi.Entities.Translate.Languages.Request.LanguagesRequest { Key = {GOOGLE_API_KEY}, Target = GoogleApi.Entities.Translate.Common.Enums.Language.English }).Data.Languages.ToList();`

I don't know if there are more languages missing in the GoogleApi.Entities.Translate.Common.Enums.Language enum.

**Comment by vivet** [comment]
I will add it.
Thanks for noticing.

**Comment by ewhoambra** [comment]
I forked and started testing the LanguagesTest, it seems that other languages were missing, and I added them in the enum, until the:

"Hebrew" language came up.

It seems that this language has two ISO-639-1 Codes "iw" and "he".

I was trying to look for a way to add multiple EnumMember values to fix this problem, but I'm stuck.

**Comment by ewhoambra** [comment]
I came up with a possible solution, but I don't think it is a good one.

Adding another value in this way seems to solve the problem and test runs successfully.

```
/// <summary>
/// Hebrew.
/// </summary>
[EnumMember(Value = "iw")]
Hebrew,

/// <summary>
/// Hebrew. (2nd ISO-631-1 code)
/// </summary>
[EnumMember(Value = "he")]
Hebrew1 = Hebrew,
```

Of course value in LanguagesTests.cs needed update from:
`Assert.AreEqual(104, languages.Count());`
to:
`Assert.AreEqual(111, languages.Count());`

**Comment by ewhoambra** [comment]
#221

**Comment by vivet** [comment]
I will check it out, within the next couple of days.
I think either the IW should be removed (its the older standard for Hebrew), or it should be named differently that Hebrew1. As far as I can see, Google will return same results no matter if you use one or the other, rendering the duplicate irrelevant, though I have investigated it fully.

**Comment by vivet** [comment]
I have merged your PR, and published an updated NuGet package, v3.10.4

**Comment by ewhoambra** [comment]
Nice! I close this issue.
